### PR TITLE
Fix errors in examples tests

### PR DESCRIPTION
**Pull request by harupy**
Signed-off-by: harupy <17039389+harupy@users.noreply.github.com>

## What changes are proposed in this pull request?

Fixes the following errors:
- https://github.com/mlflow/mlflow/pull/3759/checks?check_run_id=1484289241#step:4:1448
- https://github.com/mlflow/mlflow/pull/3759/checks?check_run_id=1484289241#step:4:955

## How is this patch tested?

GitHub Actions

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components 
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [x] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: Local serving, model deployment tools, spark UDFs
- [ ] `area/server-infra`: MLflow server, JavaScript dev server
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface 
- [ ] `area/uiux`: Front-end, user experience, JavaScript, plotting
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language 
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: examples/pytorch/BertNewsClassification/conda.yaml [hunk 1]
```diff
@@ -13,6 +13,6 @@ dependencies:
   - sklearn
   - cloudpickle==1.6.0
   - boto3
-  - transformers
+  - transformers<4.0.0
```
Comment (harupy) [inline examples/pytorch/BertNewsClassification/conda.yaml:16]: Avoid the latest version released today: https://pypi.org/project/transformers/4.0.0/

File: examples/pytorch/conda.yaml [hunk 2]
```diff
@@ -9,6 +9,6 @@ dependencies:
   - pip
   - pip:
     - torch==1.6.0
-    - torchvision==0.5.0
+    - torchvision==0.7.0
```
Comment (harupy) [inline examples/pytorch/conda.yaml:12]: `torchvision==0.7.0` should be compatible with `torch==1.6.0` according to the table in https://github.com/pytorch/vision#installation

File: examples/pytorch/BertNewsClassification/conda.yaml (hunk 1 above)
Comment (smurching) [reply examples/pytorch/BertNewsClassification/conda.yaml:16]: What's the problem with the latest `transformers`? Is there a github issue tracking the problem?

**Review by smurching — APPROVED** [review]
@harupy looks good to fix the OSS build, thanks for the PR!

File: examples/pytorch/BertNewsClassification/conda.yaml (hunk 1 above)
Comment (dbczumar) [reply examples/pytorch/BertNewsClassification/conda.yaml:16]: For posterity, I think that `!=` is generally better than `<` for examples / tests, assuming that the issue is corrected in a subsequent release.

Comment (harupy) [reply examples/pytorch/BertNewsClassification/conda.yaml:16]: @dbczumar Thanks for the advice! I'll do so from the next time:)

I investigated this issue. In transformers 4.0.0, a model returns a dict:

```
outputs = model(**inputs)  # outputs is a dict.
```

So when we unpack an model output like below, the second key (`str`) will be assigned to `pooled_output` which is supposed to be a torch tensor.

```
_, pooled_output = self.bert_model(input_ids=input_ids, attention_mask=attention_mask)
```

Comment (harupy) [reply examples/pytorch/BertNewsClassification/conda.yaml:16]: I'll fix this.

Comment (harupy) [reply examples/pytorch/BertNewsClassification/conda.yaml:16]: Opened #3774

**Pull request merged**